server/scripts/create_database.sql

```diff
@@ -21,8 +21,32 @@ CREATE TABLE clients
     language_spoken VARCHAR(120),
     english_speaker BOOLEAN,
     english_skill_level VARCHAR(120),
-    gender INT,
+    gender VARCHAR(120),
     date_of_birth DATE,
     postcode VARCHAR(20),
     referring_agency VARCHAR(120)
 );
+
+CREATE TABLE booking_status 
+(
+	booking_status_id SERIAL PRIMARY KEY,
+	status VARCHAR
+);
+	
+INSERT INTO booking_status (status) values ('waiting'), ('contacted'), ('booked'), ('declined'), ('completed'), ('cancelled');
+
+CREATE TABLE residency_status 
+(
+	residency_status_id serial primary key,
```

Comment:
```suggestion
	residency_status_id SERIAL PRIMARY KEY,
```
The `SERIAL PRIMARY KEY` above is capitalised, so there's an inconsistency. Same with varchar below and values above and below.

This is very minor and some teams won't mind or care, but let's practise being pedantic anyway 🤓

Reply:
🤓